### DISCUSSION TITLE
Linking SuperAGI with Google Spreadsheet for Cluster Analysis?"

### DISCUSSION THREAD

**Discussion by zivafgin** [TransformerOptimus/SuperAGI#332] [Q&A]
Hello everyone,
Looking to link SuperAGI to Google Spreadsheet for grouping similar data/Cluster Analysis. 
is there someone who knows if it's possible?

manny thanks,
Ziv

**Comment by neelayan7** [comment]
Hey, we are integrating with Llama Index in v0.0.7 release planned for today. This should solve your query.
